control/control.CASP.xml

```diff
@@ -463,6 +468,17 @@ defined by a role can be overridden in the next steps if necessary.&lt;/p&gt;</l
                 <module>
                     <label>Load Linuxrc Network Configuration</label>
                     <name>install_inf</name>
+                    <!-- Configuring libzypp in order to install a minimal system (no recommended packages,...) Fate#321764 -->
```

Comment:
Is the control file the correct place to set this? Maybe YaST is the proper place. I'm not saying that it's wrong but I expected this to be anywhere else :)